### PR TITLE
Ensures playspace transform is created in editor

### PR DESCRIPTION
**Pull request by Railboy**
## Overview
Creates a playspace transform and parents main camera under playspace transform in editor.

## Changes
- Fixes: #4334

### CODE REVIEW

**Comment by wiwei** [comment]
@Railboy do you know the history for why this got removed in the first place and why that's not valid?

**Comment by Railboy** [comment]
@wiwei This behavior changed when the static MixedRealityPlayspace class was introduced. At that point Playspace creation was removed from MixedRealityToolkit.EnsureMixedRealityRequirements, since it was no longer the MixedRealityToolkit instance's job to actually create the transform. But we should have kept a call to the transform getter in the instance.

File: Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs [hunk 1]
```diff
@@ -488,6 +488,11 @@ private void EnsureMixedRealityRequirements()
             // We'll enforce that here, then tracking can update it to the appropriate position later.
             CameraCache.Main.transform.position = Vector3.zero;
 
+            // This will create the playspace
+            Transform playspace = MixedRealityPlayspace.Transform;
+            // Parent the camera under the playspace
+            CameraCache.Main.transform.parent = playspace;
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs]: Looks like this should already happen in the `Transform` call. Were you hitting cases where it wasn't?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs#L35

**Pull request merged**